chroma_core/services/job_scheduler/lock_cache.py

```diff
@@ -123,9 +123,14 @@ def _decorator(func):
 
 
 def to_lock_json(lock, add_remove=LockCache.LOCK_ADD):
+    if getattr(lock.locked_item, "downcast", None) and callable(lock.locked_item.downcast):
```

Comment:
What was happening if it wasn't being downcasted? Was it throwing an error?

Reply:
It would return a different content_type_id iirc. You were seeing it on the MGT page during testing.